src/app/(BottomTabNavigation)/Profile/DeleteAccount/index.tsx

```diff
@@ -0,0 +1,46 @@
+//import { router } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import {
+  deleteCurrentUser,
+  getCurrentUserInfo,
+  signOutUser,
+} from '../../../../supabase/queries/auth';
+import styles from '../styles';
+
+function DeleteAccountScreen() {
+  const [currSession, setCurrSession] = useState<any>(null);
+  useEffect(() => {
+    getCurrentUserInfo().then(result => {
```

Comment:
Since we now throw an error if `getCurrentUserInfo` returns `null`, we don't need an else case and we can set the `result` information to a `User` type so that we guarantee that the types are correct.